### ISSUE TITLE
Build log service is not available from China

### ISSUE THREAD

**Issue by rockonedege** [netlify/build-image#161]
I have to build my own log and publish it check. If , and quite often, it failed to build, I am out of luck and have to resort to blind guesses. It wasn't like this in the beginning until some time you changed the service provider.

To be honest,  I am currently using the free service so I cannot really complain about it. However,  this might bock you from many potential paid customers from this region(me included).

Being a grateful user, I feel obliged to report it incase you are not aware.

**Comment by brycekahle** [comment]
@rockonedege Are you still unable to access build logs?

**Comment by rockonedege** [comment]
Not last time I checked one minute ago.

**Comment by brycekahle** [comment]
@rockonedege We currently use firebase for the build logs, and it looks like all Google domains are blocked in China 😢

**Comment by rockonedege** [comment]
In general, google services,CDN included, are at best slow or unreliable, if not inaccessible at all in China. Microsoft's are generally OK. AWS is in between.

**Comment by jhabdas** [comment]
According to Worldometers the [current population in China](http://www.worldometers.info/world-population/china-population/) is 1,415,971,400 and the [world population](http://www.worldometers.info/world-population/) is 7,646,606,545. Therefore, approximately 18% of the world is not able to access Netlify build logs. 哎哟!

**Comment by aicochow** [comment]
netlify blocked in china

**Comment by rockonedege** [comment]
not true. netlify is perfectly available from China. My site works like a charm. Only the build log , a google service netlify uses, is not accessible universally, which, I just found ou lately, is blocked by China Telecom, bu not by China Mobile.

So it's up to to ISP inthe end.

**Comment by jhabdas** [comment]
Netlify has their own CDN, but it runs on six different cloud providers as of mid-2018, Google [being one of them](https://www.netlify.com/blog/2018/05/14/how-netlify-migrated-to-a-fully-multi-cloud-infrastructure/).

**Comment by jhabdas** [comment]
Netlify has their own CDN, but it runs on six different cloud providers, Google [being one of them](https://www.netlify.com/blog/2018/05/14/how-netlify-migrated-to-a-fully-multi-cloud-infrastructure/). This is politics at its best here folks. And if you think build logs aren't important you've got some personal growth to do in the CI/CD department.

**Comment by rockonedege** [comment]
I actually think logs are very important and that's why I opened this ticket.

as commented by netlify earlier , it's the **Google firebase service** that was blocked, not netlify's CDN.It's a collateral damage b/w Google and the Chinese ISPs.

As there are a few competing ISPs, I am pleased to see they don't agree on this very item in the blocking lists. I'm providing this info for those who need it here.

**Comment by bcomnes** [comment]
We are looking to move away/proxy logs from firebase, but I don't have a timeline.
